pkg/identity/identity.go

```diff
@@ -53,6 +55,14 @@ type FullIdentity struct {
 	Key crypto.PrivateKey
 }
 
+// ManageablePeerIdentity is a `PeerIdentity` and it's corresponding `FullCertificateAuthority`
```

Comment:
super mega nit - "it's" should be "its"

Reply:
also, what does the addition of a ManageablePeerIdentity do, and how did we get around making changes to IDs that require CA auth before its existence?

Reply:
I'm the worst with "it's" vs "its". `ManageablePeerIdentity` provides a convenient place to add methods to make changes to identities that require the presence of its (nailed it) CA private key. Previously we were only using the `ExtraExtensions` field on the certificate structs (which is wrong) which didn't require re-signing. Now that we're using `Extensions`, we need to re-sign the certificate to add an extension.